modules/katamari/src/test/ts/atomic/api/Regex/RegexTest.ts

```diff
@@ -0,0 +1,17 @@
+import { describe } from '@ephox/bedrock-client';
+import { assert } from 'chai';
+import * as Regex from 'ephox/katamari/api/Regex';
+
+describe('Regex.escapeRegExp', () => {
```

Comment:
`describe` is used to group the tests, assertions should always exist inside a test (`it`).

- [ ] Add an `it` block to this `describe`

Reply:
It might be best to do something like this:
```
describe('Regex', () => {
  it('escapeRegExp', () => {
     // Tests and assertions
  });
});
```

Reply:
updated